packages/input-output-logger/index.js

```diff
@@ -1,21 +1,29 @@
+const omit = require('lodash/omit')
+
 module.exports = (opts) => {
   const defaults = {
-    logger: data => console.log(JSON.stringify(data, null, 2))
+    logger: data => console.log(JSON.stringify(data, null, 2)),
+    exclude: []
```

Comment:
Maybe a little bit more explicit to call this something like `omitPaths`?

Reply:
Done [here](https://github.com/middyjs/middy/pull/507/commits/cd02db468a56d2184577eaeae40e5fadb0468508)

Reply:
;)